### PR TITLE
Feat/select

### PR DESCRIPTION
**Pull request by Chzzs**
Add component called `<Test />` which is a Select component made with react-select. 
It seems more reliable than our home-brewed `<Select />` 

One major caveat: react-select does not support required attribute 
https://github.com/JedWatson/react-select/issues/3140

But this comment solves it: https://github.com/JedWatson/react-select/issues/3140#issuecomment-514754657
So I copied/pasted the code in the styleguidist. 😅

---

### next steps

- [ ] check that the to component can easily be switched
- [ ] refactor the SearchMultipleInput to use the react-select too.

### CODE REVIEW

File: src/components/Test.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
+import React from 'react';
```
Comment (bpacaud) [inline src/components/Test.js:1]: rename file "SelectBis.js" rather than Test ?

File: src/components/Test.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+```jsx
```
Comment (bpacaud) [inline src/components/Test.md:1]: rename file "SelectBis.md" rather than Test ?

File: src/components/Test.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+<div>
+<form onSubmit={e=>{e.preventDefault();console.log("hello")}} >
+
+  <Test label="test" required options={[{label:"test", value:"adadad"},{label:"hello", value:"moto"}, {label:"test", value:"coucou"},{label:"hello", value:"tzaeaze"},{label:"test", value:"rezrzerzerz"},{label:"hello", value:"vsdazeazeaze"},]} onChange={e=>console.log(e)}/>
+  <button>ok</button>
```
Comment (bpacaud) [inline src/components/Test.md:6]: remove button ?

Comment (Chzzs) [reply src/components/Test.md:6]: button is required to test the required attribute

Comment (bpacaud) [reply src/components/Test.md:6]: 👍 
You will rename Test or I can approve ?

**Pull request closed without merging**